### ISSUE TITLE
`getNodeText` is slow

### ISSUE THREAD

**Issue by xPaw** [birchill/bugsnag-zero#135]
Specifically this:

https://github.com/birchill/bugsnag-zero/blob/a854c7e76584faa6d4e3e9d014f14c241c93e266/src/interaction-breadcrumbs.ts#L56

When clicking on page background, it's trying to get text of the whole body, which isn't good on big pages.

![image](https://github.com/birchill/bugsnag-zero/assets/613331/dfe8067e-1ffe-4b92-9bfe-d5b145ba1f56)

**Comment by birtles** [comment]
Thanks for reporting this! That looks like an easy fix—just truncate before trimming whitespace. I can add the option to disable displaying the text but I'm curious if it's needed?

**Comment by birtles** [comment]
I think adding an option to the plugin might be a breaking change unless we expose separate function names or do some very clever argument sniffing so I'd probably prefer to handle that as a separate issue/patch.

**Comment by birtles** [comment]
Should be fixed in 0.6.8 now I hope.

**Comment by xPaw** [comment]
There's also string.trim() that could be used instead of regex.

**Comment by birtles** [comment]
Yeah, I have no idea why I went with that regex. I probably cribbed it from somewhere else because normally I would reach for `trim()`. Oh well, next version.

**Comment by birtles** [comment]
Fixed on `main`.
